REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.4
-Compat 0.7.15
+Compat 0.9.4
```

Comment:
As usual, thanks for all your expertise and your attention to this package and the many others.